### PR TITLE
Feature/kickstarter trajectory 2017

### PR DESCRIPTION
**Pull request by Mw1010**
Request to add kickstarter 2017 blog to the master branch

### CODE REVIEW

File: _posts/2017-10-01-Kickstarter-Trajectory-2017.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+---
+
+>This year, five talented people started the Ordina kickstarter trajectory. 
+Each of them looking for a challenge and a fresh start! 
+For some, it was a transition between school and work and Ordina handled this very well. 
+The main goal of this trajectory was to widen every student’s knowledge of the IT world. 
+They taught us the basics of every topic that’s hot at the moment. 
+This will definitely come in handy during our first project.
+
+# Kickstarter trajectory 2017
```
Comment (yannickdeturck) [inline _posts/2017-10-01-Kickstarter-Trajectory-2017.md]: This line can be removed, as it's already the title on top of the page.

File: _posts/2017-10-01-Kickstarter-Trajectory-2017.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+"The soft skill sessions were real eyeopeners and gave me a better understanding regarding introducing myself, giving presentations and the Agile methodology. 
+The introduction to microservices was very interesting since moving away from monoliths is the way to go." -- Ken
+
+"The tips received during the communication essentials sessions are a backpack full good enough to present myself in a proper way. 
+Diverse technologies used in Jworks e.g. turning a monolith to Microservices, a different type of databases like MonogDB, running your applications in docker containers and such." -- Michiel
+
+"Learning about the existence of 'Microservices' with 'monoliths'. 
+Making unit-tests to see if the code does what it should do before launching the application in production.
+And the introduction to the mongoDatabase, how to use it in the command line interface." --Jef
```
[the remark this answers is not in the document]
Comment (yannickdeturck) [reply _posts/2017-10-01-Kickstarter-Trajectory-2017.md]: Also prepend "Jef" with a space.

File: _posts/2017-10-01-Kickstarter-Trajectory-2017.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+"The communication sessions were worth their weight in gold! 
+How often do you get a chance to practice communication in the most awkward situations? 
+I guess every day, but at least in these sessions there were no real repercussions :)" -- Simon
+
+"Learning about all those different technologies. 
+There really is to much to chose from.
+But if I have to pick something, it's the Spring Boot session.
+I previously experimented with Spring Boot at my internship, where nothing seemed to make sense.
+But after this session, everything I was struggling with became clear and fell in place.
+" --Nick
```
Comment (yannickdeturck) [inline _posts/2017-10-01-Kickstarter-Trajectory-2017.md]: Prepend "Nick" with a space.

File: _data/authors.yml [hunk 4]
```diff
@@ -343,3 +343,49 @@ martijn_willekens:
...
   bio: Martijn is a frontend developer at Ordina Belgium. He primarily devotes himself to building Angular and Ionic apps written in TypeScript. ReactiveX is his favorite set of tools to work with.
+jef_willems:
+  first_name: Jef
+  last_name: Willems
+  linkedin: jef-willems-2022aa135
+  permalink: /author/jef-willems/
+  avatar: jef-willems.jpg
+  title: Java Developer
+  email: jef.willems@ordina.be
+  bio: Jef is a Junior Java Developer at Ordina in Belgium. His intrest for databases results in foccusing on the back-end part. Jef has good experience with SQL, he also starts to explore mongoDb.<br />After succeeding the Kickstart-traject 2017 he also saw that Unit testing and Microservices is something where he would like to dive deeper in.
```
[the remark this answers is not in the document]
Comment (TimDG) [reply _data/authors.yml]: - I'm not sure about the "has good experience": it implies he simply likes using it.

[the remark this answers is not in the document]
Comment (yannickdeturck) [reply _data/authors.yml]: Indeed, probably best to rephrase that part...
Something along the lines of "Jeff enjoys working with SQL"

File: _data/authors.yml [hunk 5]
```diff
@@ -343,3 +343,49 @@ martijn_willekens:
...
+michiel_willems:
+  first_name: Michiel
+  last_name: Willems
+  permalink: /author/michiel-willems/
+  avatar: michiel-willems.jpg
+  title: Java Developer
+  email: michiel.willems@ordina.be
+  linkedin: michiel-willems-21b45211a
+  bio: Michiel is a junior Java Developer at Ordina Belgium. Just graduated from kickstarter project 2017. And he's mainly focussing at back-end and databases.
```
[the remark this answers is not in the document]
Comment (TimDG) [reply _data/authors.yml]: linked the first 2 sentences: ...at Ordina Belgium, who just graduated...
Drop the "and" in the last one.

File: _data/authors.yml [hunk 6]
```diff
@@ -343,3 +343,49 @@ martijn_willekens:
...
+ken_de_becker:
+  first_name: Ken
+  last_name: De Becker
+  permalink: /author/ken-de-becker/
+  avatar: ken-de-becker.jpg
+  title: Java Developer
+  email: ken.debecker@ordina.be
+  bio: Ken is a junior Java Developer at Ordina Belgium. His main focus is on back-end development. He enjoys working with the underlying logic of applications and learning new and innovative technologies.
```
[the remark this answers is not in the document]
Comment (TimDG) [reply _data/authors.yml]: "He enjoys working with the underlying logic of applications as well as learning new and innovative technologies"

File: _posts/2017-10-01-Kickstarter-Trajectory-2017.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+  * Microservices 
+  * Agile, Scrum 
+  * DevOps, Continuous Integration
+  * DDD, BDD, TDD
+
+Rather than learning these technologies in isolation, we learned to create applications by combining them together. 
+For example, we used TypeScript, CSS and Ionic to build mobile apps, we wrote backend logic by uniting the forces of Spring Boot, Java and unit testing. 
+Thirdly we combined Angular, TypeScript, SASS, HTML and npm to create a web application. 
+
+In our projects, we learned to utilize the control system Git to collaborate with your team and share projects and code progress.
```
[the remark this answers is not in the document]
Comment (TimDG) [reply _posts/2017-10-01-Kickstarter-Trajectory-2017.md]: "with your team" sounds wrong: you're not collaborating with the reader's team.

File: _posts/2017-10-01-Kickstarter-Trajectory-2017.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+  * Cloud solutions (PaaS, IaaS, SaaS)
+  * Docker, Git
+
+* **Design principles and methodologies**
+  * Microservices 
+  * Agile, Scrum 
+  * DevOps, Continuous Integration
+  * DDD, BDD, TDD
+
+Rather than learning these technologies in isolation, we learned to create applications by combining them together.
```
Comment (TimDG) [inline _posts/2017-10-01-Kickstarter-Trajectory-2017.md]: drop "together"

File: _posts/2017-10-01-Kickstarter-Trajectory-2017.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+
+* **Design principles and methodologies**
+  * Microservices 
+  * Agile, Scrum 
+  * DevOps, Continuous Integration
+  * DDD, BDD, TDD
+
+Rather than learning these technologies in isolation, we learned to create applications by combining them together. 
+For example, we used TypeScript, CSS and Ionic to build mobile apps, we wrote backend logic by uniting the forces of Spring Boot, Java and unit testing. 
+Thirdly we combined Angular, TypeScript, SASS, HTML and npm to create a web application.
```
[the remark this answers is not in the document]
Comment (TimDG) [reply _posts/2017-10-01-Kickstarter-Trajectory-2017.md]: I'd even forego the "thirdly": there was no "firstly" or "secondly", so it makes no sense here.

File: _posts/2017-10-01-Kickstarter-Trajectory-2017.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+One thing we’ll never forget is that if push breaking changes to the master branch and break the build, we’ll have to buy “boterkoeken” for the unit! 
+Last but not least, we were introduced to the wonderful world where clean code & microservices are the heroes that kick spaghetti code & monoliths in the butt. 
+
+## Memorable moments
+
+"The soft skill sessions were real eyeopeners and gave me a better understanding regarding introducing myself, giving presentations and the Agile methodology. 
+The introduction to microservices was very interesting since moving away from monoliths is the way to go." -- Ken
+
+"The tips received during the communication essentials sessions are a backpack full good enough to present myself in a proper way. 
+Diverse technologies used in Jworks e.g. turning a monolith to Microservices, a different type of databases like MonogDB, running your applications in docker containers and such." -- Michiel
```
[the remark this answers is not in the document]
Comment (TimDG) [reply _posts/2017-10-01-Kickstarter-Trajectory-2017.md]: database (-s)

File: _posts/2017-10-01-Kickstarter-Trajectory-2017.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+
+"Learning about the existence of 'Microservices' with 'monoliths'.
```
Comment (TimDG) [inline _posts/2017-10-01-Kickstarter-Trajectory-2017.md]: with -> versus

File: _posts/2017-10-01-Kickstarter-Trajectory-2017.md [hunk 12]
```diff
@@ -8,110 +8,111 @@ category: Kickstarters
...
-"The communication sessions were worth their weight in gold! 
-How often do you get a chance to practice communication in the most awkward situations? 
+"The communication sessions were worth their weight in gold!
+How often do you get a chance to practice communication in the most awkward situations?
 I guess every day, but at least in these sessions there were no real repercussions :)" -- Simon
 
-"Learning about all those different technologies. 
-There really is to much to chose from.
+"Learning about all those different technologies.
+There really is to much too chose from.
```
Comment (yannickdeturck) [inline _posts/2017-10-01-Kickstarter-Trajectory-2017.md]: The "to" and "too" here need to be swapped :-)
too much
to chose from

File: _posts/2017-10-05-Kickstarter-Trajectory-2017.md [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+
+>This year, five talented people started the Ordina Kickstarter trajectory.
+Each of them looking for a challenge and a fresh start!
+For some, it was a transition between school and work, and Ordina handled this very well.
+The main goal of this trajectory was to widen every student’s knowledge of the IT world.
+They taught us the basics of every topic that’s hot at the moment.
+This will definitely come in handy during our first project.
+
+
+# First impressions
```
Comment (TimDG) [inline _posts/2017-10-05-Kickstarter-Trajectory-2017.md]: Please have a look at this paragraph, the first sentence and last sentence are fine. The other 4 have the same structure, which feels repetitive.

File: _posts/2017-10-05-Kickstarter-Trajectory-2017.md [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+From day one Ordina makes sure that their consultants are well equipped to work successfully.
+
+# What is the Kickstarter trajectory?
+
+The Kickstarter trajectory consisted of intensive training spanning two months.
+During this time, courses on different frontend and backend technologies, methods, tools and soft skills were given by senior consultants and external lecturers. 
+This trajectory is ideal for graduating students and people who want to make a switch to IT.
+The constant guidance and support made this a good preparation for our first project.
+
+During the fourth week we had a team building together with all the kickstarters from the different units.
```
Comment (TimDG) [inline _posts/2017-10-05-Kickstarter-Trajectory-2017.md]: Call it a "team building event" as I'm sure you didn't have a building for your team. You can also drop the "together"

File: _posts/2017-10-05-Kickstarter-Trajectory-2017.md (hunk 13 above)
Comment (TimDG) [inline _posts/2017-10-05-Kickstarter-Trajectory-2017.md]: Please have a look at this paragraph, the first sentence and last sentence are fine. The other 4 have the same structure, which feels repetitive.

**Pull request merged**